docs/stencil-docs/developing-further/customizing-emails.md

```diff
@@ -0,0 +1,22 @@
+# BigCommerce Email Templates
+
+<div class="otp" id="no-index">
+
+### On This Page
+- [BigCommerce Email Templates](#bigcommerce-email-templates)
+- [Accessing and Editing Email Templates](#accessing-and-editing-email-templates)
+
+</div>
+
+
+## BigCommerce Email Templates
+Email templates provide the framework for the store emails automatically sent to customers. These templates can be edited within the store's control panel and can also be edited locally and uploaded to a store via WebDav. For information on which user actions trigger these emails to be sent to a customer, see [Email Template Files and Snippets](https://support.bigcommerce.com/s/article/Customizing-Emails#template-files).
```

Comment:
Suggestion: 
"You can edit these templates locally or in the store's control panel, then upload them to the store via WebDav."